src/to_vec.jl

```diff
@@ -55,11 +55,19 @@ end
 
 function to_vec(x::Symmetric)
     function Symmetric_from_vec(x_vec)
-        return Symmetric(reshape(x_vec, size(x)))
+        return Symmetric(reshape(x_vec, size(x)), Symbol(x.uplo))
```

Comment:
Why do we need the Symbol conversion here (same below)?

Reply:
Even though `uplo` is internally a char, the `Symmetric(...)` constructor is only defined for `uplo::Symbol`. The alternative to this approach is to use the constructor with types: `typeof(x)(..., x.uplo)`.

Reply:
Oh, wild. That seems odd, but maybe there's a reason for it. Thanks for explaining 😊

Reply:
I switched over to the type constructor.

Reply:
Yeah, there's some strangeness around `Symmetric` and `Hermitian`. 🤷‍♂️ No problem!